### ISSUE TITLE
QUESTION: how to allow dokku app to reach localhost's post

### ISSUE THREAD

**Issue by ohld** [dokku/dokku#4119]
Hi! 

I have a Datagog Agent on host machine running on port `http://localhost:8126`. I want my Dokku app to have access to this Agent. 

I saw new networking possibilities in new Dokku version (http://dokku.viewdocs.io/dokku/networking/network/) but I don't know what exactly do I need. This is maybe a really dumb question. It would be awesome if you can point me on what should I do the best.

I've googled a lot on this and seems like I need to join my dokku app to `host` network - and then the Agent's `http://localhost:8126` will be accessible from Dokku app (which is running in docker obviously).

I just updated dokku via `dokku-update` and my docker is:
```
~ docker -v
Docker version 19.03.6, build 369ce74a3c

~ dokku -v
dokku version 0.21.4
```

**Comment by josegonzalez** [comment]
This is likely the correct way to do it: https://stackoverflow.com/a/31328031/1515875

Feel free to join us on slack to ask further questions :)

**Comment by hiepxanh** [comment]
I suggest you using datalog Agent on a container, then you can point from this container to other container easily

**Comment by josegonzalez** [comment]
The following has been confirmed working on my end (and seems to have been released in [20.10.0](https://docs.docker.com/engine/release-notes/#20100)):

```
jose@dc1:~$ docker --version
Docker version 20.10.2, build 2291f61

jose@dc1:~$ docker run --rm --add-host host.docker.internal:host-gateway alpine ping host.docker.internal
PING host.docker.internal (172.17.0.1): 56 data bytes
64 bytes from 172.17.0.1: seq=0 ttl=64 time=0.535 ms
64 bytes from 172.17.0.1: seq=1 ttl=64 time=0.421 ms
64 bytes from 172.17.0.1: seq=2 ttl=64 time=0.323 ms
^C
--- host.docker.internal ping statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max = 0.323/0.426/0.535 ms
```

You can therefore use `dokku docker-options:add $APP deploy,run "--add-host host.docker.internal:host-gateway"` to add an alias named `host.docker.internal` to the host for accessing various ports.
